ni/panels/v1/streamlit_panel_configuration.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package ni.panels.v1;
+
+option cc_enable_arenas = true;
+option csharp_namespace = "NationalInstruments.Panels.V1";
+option go_package = "panelsv1";
+option java_multiple_files = true;
+option java_outer_classname = "StreamlitPanelConfigurationProto";
+option java_package = "com.ni.panels.v1";
+option objc_class_prefix = "NIPS";
+option php_namespace = "NI\\Panels\\V1";
+option ruby_package = "NI::Panels::V1";
+
+// Pack this into PanelService.StartPanelRequest.panel_configuration to start a Streamlit panel.
+message StreamlitPanelConfiguration 
+{
+  // Absolute path of the streamlit script file on disk, or network path to the file. This must end in ".py".
+  string panel_script_path = 1;
+
+  // Path to the python interpreter to use for running the streamlit script.
+  string python_path = 2;
```

Comment:
Would it make sense to be more explicit here and call this `python_interpreter_path` or `python_interpreter_url`?

Reply:
Explicit `python_interpreter_path` makes sense